aruba.gemspec

```diff
@@ -45,11 +46,9 @@ Gem::Specification.new do |spec|
   spec.rubygems_version = '>= 1.6.1'
   spec.required_ruby_version = '>= 2.4'
 
-  spec.files = `git ls-files -z`.split("\x0").reject do |f|
-    f.match(%r{^(test|spec|features)/})
-  end
+  spec.files = Dir['{docs,fixtures,lib}/**/*', 'CHANGELOG.md', 'LICENSE', 'README.md']
```

Comment:
Please add a `.exclude(*File.read('.gitignore').split)` like you did in for cucumber-rails and gir_fii.

Reply:
Aw, crap. Right!

Reply:
Should've done earlier but done now!
Thanks for the review \o/